helpers.js

```diff
@@ -3,21 +3,23 @@
 const fs = require('fs');
 const Path = require('path');
 
-let helpers = [];
+const helpers = [];
 
 // Load helpers
-fs.readdirSync(Path.join(__dirname, 'helpers')).forEach(filename => {
-    if (!fs.lstatSync(Path.join(__dirname, 'helpers', filename)).isDirectory()) {
-        helpers = helpers.concat(require('./helpers/' + filename));
-    }
-});
+const notDeprecated = fs.readdirSync(Path.join(__dirname, 'helpers'));
 
+for (let i = 0; i < notDeprecated.length; i++) {
```

Comment:
Not important to refactor this file. This only gets loaded once, and the incremental gain of using a plain for loop is negligible compared to loading files from the file system. Let's revert the changes to this file.

Reply:
fixed.